api/features/serializers.py

```diff
@@ -98,6 +98,10 @@ class FeatureQuerySerializer(serializers.Serializer):
         required=False,
         help_text="Integer ID of the environment to view features in the context of.",
     )
+    state_enabled = serializers.BooleanField(
+        allow_null=True, required=False, default=None
+    )
+    state_search = serializers.CharField(required=False, default=None)
```

Comment:
Let's add some `help_text` to these so the FE knows what they are for (it will show up in the api docs)

Reply:
Good idea. Have added locally.